### ISSUE TITLE
Analysis report part 1 feedback

### ISSUE THREAD

**Issue by ugurdogrusoz** [onurkulak/Syracuse-group-27#3]
Overall your report is fine. But reading it I am already concerned about the complex game logic that might lead to problems during design and implementation. So many different screens/maps! Can you perhaps leave out some of them?
Other problems/suggestions are marked in the attached report.
[27.analysis.1.pdf](https://github.com/onurkulak/Syracuse-group-27/files/545815/27.analysis.1.pdf)

**Comment by onurkulak** [comment]
Yes this is true hocam, game and the report seems complicated at first look but I think this is usually the case with the strategy/board games, they have lots of rules and seems complicated during game design process; but  since our game is not action based like the other games graphical components will be easier to code for us during the later stages of the project. Also the screens/maps in the game will be mostly just static pictures and texts so I think we can handle those.

**Comment by ugurdogrusoz** [comment]
@onurkulak OK, I am just making sure you know what you're into :)

**Comment by ugurdogrusoz** [comment]
Did you guys tell me about the changes you had to make after the deadline? Even if so, you should have specified what exactly changed in your commit log. Please list any changes here.

**Comment by ugurdogrusoz** [comment]
Also, why on earth would you name part 1 and 2 of your documents like that (differs by a blank space only!)? Looks like this was fixed with the revised version of the document though.

**Comment by ghost** [comment]
We added the following functions to the object-class model: 

class name: function names

region: listNeighbors()
BattleManager: chooseArmy(), calculateResult(), createReport()
SicilyLayerManager: update(), chooseProvince()
CityLayerManager: chooseBuildingCategory(), chooseBuilding(), isValidPosition(),placeBuildingPosition()

Other than functions, we change "Use Case Diagram" title to "Use Case Model".

**Comment by ghost** [comment]
AnalysisReport: The first file we uploaded(until 5.2).
Analysis Report: Complete report.
AnalysisReport2: Complete report with changes.

**Comment by ugurdogrusoz** [comment]
Your poor naming of files to maximize confusion continues!
The difference in the names of the two design reports (high and low) level is only in the extension!
